extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/sql/MSQTaskQueryMaker.java

```diff
@@ -281,6 +282,16 @@ public QueryResponse<Object[]> runQuery(final DruidQuery druidQuery)
                .tuningConfig(new MSQTuningConfig(maxNumWorkers, maxRowsInMemory, rowsPerSegment, indexSpec))
                .build();
 
+    Map<String, Object> context = new HashMap<>();
+    if (plannerContext.getLookupsToLoad() != null) {
```

Comment:
the planner context should give a `LookupLoadingSpec` which should be used to populate the context, rather than relying on null and empty values.